lib/widgets/rules_buttons.dart

```diff
@@ -43,8 +42,7 @@ class RulesChangingButtonsState extends State<RulesChangingButtons> {
                   setState(() {
                     selectedValue = value ?? '3';
                     inRowValue = int.tryParse(value ?? '0') ?? 0;
```

Comment:
Why value is string? Can we make it int, so that we do not need to parse?

Reply:
We have universal dropdown widget, that takes strings as an argument and returns string. And then we change this string to type we want (in this case int, in another bool).